sources/packages/backend/apps/db-migrations/src/migrations/1705441135641-RemovePDVerifiedColumn.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class RemovePDVerifiedColumn1705441135641 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Remove-StudentPDVerified.sql", "Student"),
```

Comment:
Not part of this PR but I believe that we should simplify the DB migrations following the below convention, adding the "rollback" as a prefix.
- "Add-something-new-to-db"
- "Rollback-add-something-new-to-db"

Sometimes the rollback file name can be misleading.
@dheepak-aot @andrepestana-aot @guru-aot @sh16011993 @lewischen-aot thoughts?

Reply:
Yes, maybe we should be even more generic and use like "change" and "rollback" because sometimes in the db change we do an update or delete.

Reply:
I agree @andrewsignori-aot. Adding a `rollback` prefix simplifies finding the revert script.

Reply:
> Yes, maybe we should be even more generic and use like "change" and "rollback" because sometimes in the db change we do an update or delete.

Do you mean to use  "change" and "rollback" for the `down` script?